tests/client/amo/components/TestLanguagePicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+
+import LanguagePicker from 'amo/components/LanguagePicker';
+import { getFakeI18nInst } from 'tests/client/helpers';
+import I18nProvider from 'core/i18n/Provider';
+
+
+describe('LanguagePicker', () => {
+  function renderLanguagePicker({ ...props }) {
+    return findRenderedComponentWithType(renderIntoDocument(
+      <I18nProvider i18n={getFakeI18nInst()}>
```

Comment:
I think we've been moving away from setting up these providers in tests but that could just be me. In the end we just want to test the base component though and it accepts `i18n` as props so I don't think there's anything wrong with skipping the provider.